lib/node_modules/@stdlib/blas/base/cscal/README.md

```diff
@@ -0,0 +1,243 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# cscal
+
+> Scales a complex single-precision floating-point vector by a single-precision floating point constant.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var cscal = require( '@stdlib/blas/base/cscal' );
+```
+
+#### cscal( N, cx, x, strideX )
```

Comment:
This API seems confused. It should be

```
cscal( N, ca, cx, strideX )
```

where `ca` is [`Complex64`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/complex/float32), not a `Complex64Array`.